src/server/server_family.cc

```diff
@@ -570,7 +570,7 @@ error_code ServerFamily::LoadRdb(const std::string& rdb_file) {
   if (res) {
     io::FileSource fs(*res);
 
-    RdbLoader loader(script_mgr());
+    RdbLoader loader{&service_};
```

Comment:
I think this is part of the next commit right?

Reply:
Yes, please ignore this one line 🙂  I will remove it when I add fixes to this